lib/Widget/evaluationForms/previsitForm/generalForm.dart

```diff
@@ -0,0 +1,2653 @@
+                                        color: Color(0xFF2ED47A),
+                                        child: Text(
+                                          'ถัดไป',
+                                          style: TextStyle(fontSize: 18),
+                                        ),
+                                        onPressed: () {
+                                          Map<String, dynamic> formDataToDB = {
+                                            'hn': _hn,
+                                            'an': _an,
+                                            'patientName': _patientName,
+                                            'patientSurname': _patientSurname,
+                                            'dod': _dob,
+                                            'operationDate': _operationDate,
+                                            'gender': _gender,
+                                            'ward': _ward,
+                                            'operation': _operation,
+                                            'diagnosis': _diagnosis,
+                                            'consentSigned': _consentSigned,
+                                            'preMedication': _preMedication,
+                                            'typeOfAnesthesia':
+                                                _typeOfAnesthesia,
+                                            'previousIllness': _previousIllness,
+                                            'drugUsed': _drugUsed,
+                                            'asaClass': _asaClass,
+                                            'bw': _bw,
+                                            'high': _high,
+                                            'weight': _weight,
+                                            'previousSurgery': _previousSurgery,
+                                            'antiPlateletReason':
+                                                _antiPlateletReason,
+                                            'antiPlateletDays':
+                                                _antiPlateletDays,
+                                            'allergyMedication':
+                                                _allergyMedication,
+                                            'allergyLatex': _allergyLatex,
+                                            'psychologicalStatus':
+                                                _psychologicalStatus,
+                                            'drugAndSubstance':
+                                                _drugAndSubstance,
+                                            'sleepDisorder': _sleepDisorder,
+                                            'sleepDisorderDuration':
+                                                _sleepDisorderDuration,
+                                            'sleepDisorderDurationAvg':
+                                                _sleepDisorderDurationAvg,
```

Comment:
Please help calculate the BMI value and map it with this form. Bc this form has the BMI field but I think we should calculate it instead of the user calculating by themself. 
<img width="383" alt="Screenshot 2564-03-22 at 22 05 40" src="https://user-images.githubusercontent.com/57939595/112011376-cdbd8800-8b5a-11eb-9c18-7ef7e3c94fc1.png">